scripts/bulk-mailer/index.js

```diff
@@ -42,8 +36,25 @@ const logMock = {
  *   as the output directory.
  * @param {String} emailOutputDirname if `shouldSend` is falsy, write emails to this directory.
  *   See `shouldSend`
+ * @param {Boolean} useVerboseLogging if `true`, print info/trace messages to the console
  */
-module.exports = async function (userRecordsFilename, mailerMethodName, batchSize, batchDelayMS, shouldSend, emailOutputDirname) {
+module.exports = async function (userRecordsFilename, mailerMethodName, batchSize, batchDelayMS, shouldSend, emailOutputDirname, useVerboseLogging) {
+
+  const logMock = {
+    amplitudeEvent: () => {},
+    error: console.error,
+    info (msg) {
+      if (useVerboseLogging) {
+        console.info(msg)
```

Comment:
Can you wrap the `msg` to `JSON.stringify(msg)` so the messages are line oriented? Otherwise, it's an unparseable mix of single and multi-line messages.